src/prodbg/main/src/windows.rs

```diff
@@ -150,6 +175,7 @@ impl Windows {
                   sessions: &mut Sessions,
                   view_plugins: &mut ViewPlugins,
                   backend_plugins: &mut BackendPlugins) {
+        sleep(Duration::from_millis(100));
```

Comment:
The point of this branch is to have the changes here before merging it into master (I have filed a bunch of bugs I found when testing) that I would like to be fixed before merge to master. I will also have some more people testing it. You can of course have the 100 ms sleep around until we decide to merge to master.